charts/civil-service/values.preview.template.yaml

```diff
@@ -38,6 +38,7 @@ java:
     FEIGN_CLIENT_CONFIG_REMOTEREPOSITORYSERVICE_URL: http://${SERVICE_NAME}-camunda/engine-rest/
     SERVICE_REQUEST_UPDATE: https://${SERVICE_NAME}.preview.platform.hmcts.net/service-request-update
     SERVICE_REQUEST_UPDATE_CLAIM_ISSUED: https://${SERVICE_NAME}.preview.platform.hmcts.net/service-request-update-claim-issued
+    WA_TOGGLE: WA 4
```

Comment:
You dont need this entry here, as its same as values.yaml

Reply:
removed from here